kong/plugins/opentelemetry/schema.lua

```diff
@@ -50,6 +50,8 @@ return {
         { send_timeout = typedefs.timeout { default = 5000 } },
         { read_timeout = typedefs.timeout { default = 5000 } },
         { http_response_header_for_traceid = { type = "string", default = nil }},
+        { header_type = { type = "string", required = true, default = "preserve",
```

Comment:
This new field needs to be added [here](https://github.com/Kong/kong/blob/a0210e0b498c3f4240f833c57149ec10460f016a/kong/clustering/compat/removed_fields.lua#L66-L68) to keep compatibility with older data planes.

Reply:
Most recent commit took care of this, thanks for calling it out!